### DISCUSSION TITLE
Default or flagship destop

### DISCUSSION THREAD

**Discussion by P1tape** [SpiralLinux/SpiralLinux-project#263] [Q&A]
I like Gecko linux very much because it provides one ISO based on OpenSuse Tumbleweed. I want to try Spiral Linux to get more familiar with Debian. What is the default or 'flagship' desktop for Spiral Linux? Many desktops ISOs are proposed with no indication which one is the 'main' environment.

**Comment by geckolinux** [comment]
Hi there, there is no default. A primary goal of SpiralLinux is to offer as many desktop environments as possible for different needs and preferences.

**Reply by P1tape** [reply]
Thanks and congratulations for your fine job!

**Reply by m1k3s0** [reply]
Out of curiosity, which DE do you prefer as your everyday driver?

**Reply by P1tape** [reply]
Cinnamon when I want peace of mind. Plasma when I feel tinkering a bit more...

**Reply by m1k3s0** [reply]
What about you @geckolinux?

**Reply by geckolinux** [reply]
@m1k3s0 Definitely Cinnamon on almost all machines. Probably Mate if the machine has extremely limited resources (single-core Celeron processor, 1 GB RAM, etc.)

**Reply by m1k3s0** [reply]
Makes sense!

**Reply by m1k3s0** [reply]
@geckolinux Also, not sure if you tried this icon theme yet or not but the Obsidian icons (obsidian-icon-theme) go great with the default wallpaper.
![Obsidian icon theme in Spiral Linux](https://files.catbox.moe/8otyhk.png)

**Comment by banzaigtv** [comment]
GNOME seems to be the best edition.
![Screenshot from 2023-06-22 23-03-28](https://github.com/SpiralLinux/SpiralLinux-project/assets/80161570/dd1cafa2-a057-4b01-964a-b35b7ced99be)

**Comment by zparihar** [comment]
I think KDE is the best Spiral version ;-)
Because I love KDE, and your defaults were awesome!
